editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -417,6 +425,16 @@ function turnCodeSnippetIntoSourceMapNodes(
     } else if (strippingDefault) {
       defaultFound = true
     } else {
+      // When there's an `export default function()` we strip off the `export default`, but without
+      // that it becomes an invalid chunk of JavaScript without a name so add in a default one.
+      if (
+        exportFound &&
+        defaultFound &&
+        currentStringBuffer === 'function' &&
+        FunctionStart.test(sourceCode.substr(i))
+      ) {
+        currentStringBuffer = 'function defaultFunctionName'
```

Comment:
Should this default name be a constant? Is it worth making this function name more obscure / specific to utopia to prevent any potential name clash (e.g. `utopia_defaultFunctionName`)?

Reply:
Genuine question, because I'm not seeing this function name referenced anywhere, so I'm not sure of the scope of any potential issues here.

Reply:
Having this name gets this to just slide under the door of Babel and hasn't needed to be used so far, but there's certainly no downside in making the name less collision prone.

Reply:
Fixed in c9bb0678.